<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Youtube-dl GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/new/img/icone.png</normaloff>:/new/img/icone.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Youtube-dl GUI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lien Youtube :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lienLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downloadPushButton">
        <property name="text">
         <string>Télécharger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopPushButton">
        <property name="text">
         <string>Arrêter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dossier de destination :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="destinationLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="openPushButton">
        <property name="text">
         <string>Ouvrir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="optionsCheckBox">
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="optionsGroupBox">
      <property name="title">
       <string>Options</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="titleCorrectorCheckBox">
         <property name="toolTip">
          <string>Ce n'est sûrement pas parfait mais ça peut être bien pratique.</string>
         </property>
         <property name="text">
          <string>Corriger le titre</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="videoFormatCheckBox">
           <property name="text">
            <string>Format vidéo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="videoCodeSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showVideoFormatsPushButton">
           <property name="text">
            <string>Afficher les formats disponibles</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="audioOnlyCheckBox">
           <property name="text">
            <string>Seulement l'audio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="audioFormatLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Format de l'audio. MP3 par défaut</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="subsCheckBox">
           <property name="text">
            <string>Télécharger sous-titre</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="subsLineEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Laisser vide pour télécharger la langue par défaut.</string>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="placeholderText">
                <string>Langue par défaut si vide</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="showSubsPushButton">
               <property name="text">
                <string>Afficher les sous-titres disponibles</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="autosubCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>sous-titres générés automatiquement</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="playlistCheckBox">
         <property name="text">
          <string>Télécharger la playlist complète</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="downloadingLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="globalProgressBar">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%v/%m</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="logCheckBox">
      <property name="text">
       <string>Afficher les détails</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="outputPlainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions_avanc_es">
    <property name="title">
     <string>Options avancées</string>
    </property>
    <addaction name="actionAlertes"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionMettre_jour_youtube_dl"/>
    <addaction name="actionA_Propos_de_Youtube_dl"/>
   </widget>
   <addaction name="menuOptions_avanc_es"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAlertes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alertes</string>
   </property>
  </action>
  <action name="actionA_Propos_de_Youtube_dl">
   <property name="text">
    <string>A Propos de Youtube-dl</string>
   </property>
  </action>
  <action name="actionMettre_jour_youtube_dl">
   <property name="text">
    <string>Mettre à jour youtube-dl</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
